src/site/content/en/blog/introducing-aurora/index.md

```diff
@@ -0,0 +1,225 @@
+An aurora is a natural light display that shimmers in the sky. As we are trying
+to help user experiences built with frameworks shimmer and shine, we thought
+this name was an appropriate choice.
+
+{% Img src="image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/lxaUR7Za1DoQMJuJEZHP.png",
+alt="Aurora logo", width="800", height="450" %}
+
+In the coming months, we'll be sharing a lot more detail on Aurora. This is a
+collaboration between a small team of Chrome engineers (internally codenamed
+WebSDK) with experience working on front-end stacks, and framework and tooling
+authors. Our goal is to deliver the best user-experience possible for production
+apps regardless of the browser you're rendering in.
+
+## What is our strategy?
+
+At Google, we've learned a lot while using frameworks and tools to build and
+maintain large scale web applications such as Google Search, Maps, Image Search,
+Google Photos etc. We discovered how frameworks can play a crucial role in
+predictable app quality by providing **strong defaults** and **opinionated
+tooling**.
+
+Frameworks have a unique vantage point for influencing both DX and UX as they
+span the entire system: the client and the server, the development and
+production environments, and they integrate tools such as compiler, bundler,
+linter etc.
+
+<figure class="w-figure">
+  {% Img src="image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/UuOwHlcV76N8BwboHBVy.png", alt="Chart that shows
+  common tooling in frameworks", width="800", height="450" %}
+  <figcaption class="w-figcaption">
+    Common tooling used by framework developers
+  </figcaption>
+</figure>
+
+When solutions are baked into the framework, teams of developers can use these
+solutions and focus their time on what matters most to the product -- shipping
+great features for users.
+
+While we work to improve tools that live in every layer of the stack, frameworks
+such as Next.js, Nuxt.js, and Angular CLI, manage every step of an application's
+lifecycle. For this reason, and the fact that React adoption is the largest
+within the core UI framework ecosystem, most of our optimizations have begun as
+a proof of concept within Next.js before expanding to the rest of the ecosystem.
+
+Aurora supports success at scale by bringing solutions to the _right layer_ of
+popular tech stacks. By bridging the gap between browsers and frameworks, it
+enables high-quality to be a side-effect of building for the web while acting as
+a feedback loop to improve the web platform.
+
+{% Aside %}
+Check out the [Conformance](/conformance) post to learn more about a key component of
+our strategy.
+{% endAside %}
+
+## What is our work process?
+
+Our principles for how Aurora bridges browsers and the developer ecosystem are:
+humility, curiosity, scientific enquiry and pragmatism. We work with framework
+authors on improvements, collaborate with the community and do our due diligence
+prior to landing any changes.
+
+{% Img src="image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/oRxGw4dKoQJRhke09t86.png",
```

Comment:
can you update the image?
screenshot of this:
https://docs.google.com/presentation/d/16n6ywPHIJm4wckZVLTsqswbD2LEYdPFj-frFX5Q1iSg/edit#slide=id.gd8b919ad07_0_142

Reply:
Updated